src/Analyzer.Cli.FunctionalTests/Tests/ToTestSeparateParametersFile/TemplateWithSeparateParametersFile.bicep

```diff
@@ -0,0 +1,7 @@
+﻿param infra object
```

Comment:
Would you be able to write the equivalent IaC file in ARM and add it here? I can help if you would like :)

Reply:
Not really sure if I understand correctly, do you want the Bicep file in ARM in the same location?

Reply:
Sorry for not being clear. What I meant was:
1. Create a file called "...nctionalTests/Tests/ToTestSeparateParametersFile/TemplateWithSeparateParametersFile.json"

2. Inside the file, write:
{
  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
  "contentVersion": "1.0.0.0",
  "parameters": {
    "infra": {
      "type": "object",
      "metadata": {
        "description": "Location for the resource."
      }
    }
  },
  "resources":[
    {
      "type": "Microsoft.Insights/components",
      "apiVersion": "2020-02-02",
      "name": "test",
      "location": "[parameters('infra').environment.location]",
      "kind": "other",
    }
    ]
}

Reply:
Ah I see, added in [TemplateWithSeparateParametersFile](https://github.com/Azure/template-analyzer/pull/297/commits/9967bff6ea59906053eff3b4dba913abf1ce9768)